plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/messages/ModelResources_ja.properties

```diff
@@ -76,32 +79,66 @@ model_jdbc_None = \u306A\u3057
 model_jdbc_precision = \u7CBE\u5EA6
 model_jdbc_Procedure = \u30D7\u30ED\u30FC\u30B8\u30E3
 model_jdbc_rename_object = \u30AA\u30D6\u30B8\u30A7\u30AF\u30C8\u306E\u540D\u524D\u3092\u5909\u66F4
-model_jdbc_repeatable_read = \u7E70\u308A\u8FD4\u3057\u8AAD\u307F\u8FBC\u3080
+model_jdbc_read_committed = READ COMMITTED
+model_jdbc_read_uncommitted = READ UNCOMMITTED
+model_jdbc_repeatable_read = REPEATABLE READ
```

Comment:
IMHO, The transaction isolation levels no-need to translate.
I have written in English to make it clear that there is no need to translate.

Please see MySQL Japanese docs. These words were not translated.

> https://dev.mysql.com/doc/refman/8.0/ja/innodb-transaction-isolation-levels.html